src/NzbDrone.Core/Download/Clients/Transmission/Transmission.cs

```diff
@@ -204,24 +212,14 @@ private ValidationFailure TestConnection()
         {
             try
             {
-                var versionString = _proxy.GetVersion(Settings);
-
-                _logger.Debug("Transmission version information: {0}", versionString);
-
-                var versionResult = Regex.Match(versionString, @"(?<!\(|(\d|\.)+)(\d|\.)+(?!\)|(\d|\.)+)").Value;
-                var version = Version.Parse(versionResult);
-
-                if (version < new Version(2, 40))
-                {
-                    return new ValidationFailure(string.Empty, "Transmission version not supported, should be 2.40 or higher.");
-                }
+                return ValidateVersion();
             }
             catch (DownloadClientAuthenticationException ex)
             {
                 _logger.Error(ex, ex.Message);
                 return new NzbDroneValidationFailure("Username", "Authentication failure")
                 {
-                    DetailedDescription = "Please verify your username and password. Also verify if the host running Sonarr isn't blocked from accessing Transmission by WhiteList limitations in the Transmission configuration."
+                    DetailedDescription = string.Format("Please verify your username and password. Also verify if the host running Sonarr isn't blocked from accessing {0} by WhiteList limitations in the {0} configuration.", Name)
```

Comment:
I'm tempted to suggest moving this whole method to the concrete class, but perhaps that is overkill and isn't very DRY.